listeners/measured.go

```diff
@@ -27,14 +27,14 @@ func (l *stateAwareMeasuredListener) Accept() (c net.Conn, err error) {
 	sac, _ := c.(*measured.Conn).Conn.(WrapConnEmbeddable)
 	return &wrapMeasuredConn{
 		WrapConnEmbeddable: sac,
-		Conn:               *c.(*measured.Conn),
+		Conn:               c.(*measured.Conn),
```

Comment:
https://github.com/getlantern/http-proxy-lantern/blob/master/listeners/preprocessor.go
and
https://github.com/getlantern/http-proxy/tree/master/listeners